doc/sign-package.md

```diff
@@ -159,3 +159,92 @@ If the host arguments are not allowed to be passed then any specified within the
 If any host environment variables are configured as available within the SGX enclave, then this command will pass them though. Enclave specific environment variables will be added once Mystikos transfers control to the enclave.
 
 You then can ship your application in the form of this single exectuable to your desired platform.
+
+#### Packaging with EXT2 
+
+##### Signing EXT2 image
```

Comment:
One thing to note is that you dont have to sign an ext2 fs in order for it to be loaded.

Reply:
Added a note in the `Packaging with EXT2 image` section.